bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Title.java

```diff
@@ -0,0 +1,58 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2022 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import org.osgi.annotation.versioning.ConsumerType;
+
+import com.adobe.cq.wcm.core.components.models.Component;
+
+// Taken from site title core component to add localisation
+/**
+ * Defines the {@code Title} Sling Model used for the {@code /apps/core/wcm/components/title} component.
+ *
+ * @since com.adobe.cq.wcm.core.components.models 11.0.0
+ */
+@ConsumerType
+public interface Title extends Component {
```

Comment:
Why we haven't re-used the interface from WCM ? Why did we have to create one ? Ideally, only implementation should be forms specific due to localization

Reply:
@rismehta I had added it as I was not providing implementations for Links in the TitleImpl. Sure, I have removed it and I am updating the PR.